### PR TITLE
Add parsing of datetime as a result for assesment performed

### PR DESCRIPTION
**Pull request by mayerm94**
See QDM5.02 3.1.3 Assessment Performed Attributes table https://ecqi.healthit.gov/system/files/qdm_5_02_draft_2016nov30_authortimeerrata_508_compliant_v2.pdf

### CODE REVIEW

**Comment by mayerm94** [comment]
Can test with the bonnie cql4bonnie_595 branch. Using 
[TESTCMS9_GA_v5_1_Artifacts (1).zip](https://github.com/projecttacoma/cql_qdm_patientapi/files/973449/TESTCMS9_GA_v5_1_Artifacts.1.zip) To test. Notice, this does not fix the error that will pop up saying that durationBetween cannot be called on undefined. That error will persist until there is a data criteria that has the dateTime result that duration between can be called on. Create a patient with an Assessment Performed Data Criteria (TEST estimated due date) and add a result date to the criteria

File: app/assets/javascripts/datatypes/assessment.js.coffee [hunk 1]
```diff
@@ -51,14 +51,16 @@ class CQL_QDM.AssessmentPerformed extends CQL_QDM.QDMDatatype
...
-  @returns {Code|Int}
+  @returns {Code|Int|Date}
   ###
   result: ->
     if @_result
       if @_result.codes?
         code_system = @_result.codes[Object.keys(@_result.codes)[0]]
         code = @_result.codes[code_system]
         new cql.Code(code, code_system)
+      else if @_result.units == "UnixTime" # A PhysicalQuantity with unit UnixTime is a TimeStamp
```
Comment (holmesie) [inline app/assets/javascripts/datatypes/assessment.js.coffee]: What in the world is this...?

Comment (mayerm94) [inline app/assets/javascripts/datatypes/assessment.js.coffee]: QDM_5.02 says that AssessmentPerformed _can_ have result type of datetime. (Used only in the datacriteria for estimated due date).

Comment (c-monkey) [inline app/assets/javascripts/datatypes/assessment.js.coffee]: See https://github.com/projecttacoma/cql_qdm_patientapi/commit/94b62a0d597be46bfe3271949eceea10e395ce20

Comment (mayerm94) [inline app/assets/javascripts/datatypes/assessment.js.coffee]: Setting the unit to UnixTime was a way to overload the scalar type

**Pull request merged**